loopy/symbolic.py

```diff
@@ -2009,17 +2029,22 @@ def get_access_range(domain, subscript, assumptions=None, shape=None,
 
 class BatchedAccessRangeMapper(WalkMapper):
```

Comment:
This is now kind of mis-named. How about
```suggestion
class BatchedAccessMapMapper(WalkMapper):
```

Reply:
Agreed 2367c440888b3c3fdad892bbb640630c52010899